src/js/01-nav.js

```diff
@@ -133,7 +133,7 @@
   }
 
   // hide left nav on top level landing page
```

Comment:
It should not be necessary to use JavaScript to hide the nav on the top level landing page. You can simply remove it from the template (and adjust the CSS if necessary).

Reply:
@rakeshAlgo please update this PR with the (new) landing page template for top-level page when it's available.

Reply:
Hi @amarantha-k ,
Please merge this PR, in the next PR I"ll remove hide nav script for top-level page. 
And will create a new landing page without the navigation for the top-level landing page.